package/src/components/Message/hooks/useMessageActions.tsx

```diff
@@ -307,7 +307,7 @@ export const useMessageActions = <
   };
 
   return {
-    blockUser,
+    banUser,
```

Comment:
Isn't this something that could implicitly be a breaking change for users of our SDK ? Should it be mentioned somewhere ?

Reply:
hey @khushal87, I share @isekovanic concerns too. This is a breaking change that might impact customer integrations.

Reply:
Done